benchmark/index.js

```diff
@@ -87,6 +92,42 @@ test('setup', (t) => {
   t.end()
 })
 
+test('add a bunch of messages', async (t) => {
+  const sbot = SecretStack({ appKey: caps.shs })
+    .use(require('../'))
+    .call(null, { keys, path: dirAdd })
+
+  let state = validate.initial()
+  for (var i = 0; i < 1000; ++i) {
+    state = validate.appendNew(state, null, keys, { type: 'tick', count: i }, Date.now())
+  }
+
+  const ended = DeferredPromise()
+  const start = Date.now()
+
+  pull(
+    pull.values(state.queue.map(x => x.value)),
```

Comment:
To measure only sbot.db.add, could we move this array map to up there before the `start`? Like `const vals = state.queue.map(x => x.value)`. It's probably not too many milliseconds, but better to be sure